app/Nova/OAuth2Client.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Nova;
+
+use Illuminate\Http\Request;
+use Laravel\Nova\Fields\Boolean;
+use Laravel\Nova\Fields\DateTime;
+use Laravel\Nova\Fields\ID;
+use Laravel\Nova\Fields\Text;
+use Laravel\Passport\Client;
+
+class OAuth2Client extends Resource
+{
+    /**
+     * The model the resource corresponds to.
+     *
+     * @var string
+     */
+    public static $model = Client::class;
+
+    /**
+     * The single value that should be used to represent the resource when being displayed.
+     *
+     * @var string
+     */
+    public static $title = 'name';
+
+    /**
+     * The columns that should be searched.
+     *
+     * @var array<string>
+     */
+    public static $search = [
+        'id',
+        'name',
+    ];
+
+    /**
+     * Get the displayable label of the resource.
+     */
+    public static function label(): string
+    {
+        return 'OAuth2 Clients';
+    }
+
+    /**
+     * Get the displayble singular label of the resource.
+     */
+    public static function singularLabel(): string
+    {
+        return 'OAuth2 Client';
+    }
+
+    /**
+     * Indicates if the resource should be displayed in the sidebar.
+     *
+     * @var bool
+     */
+    public static $displayInNavigation = false;
+
+    /**
+     * Get the fields displayed by the resource.
+     *
+     * @param \Illuminate\Http\Request  $request
+     *
+     * @return array<\Laravel\Nova\Fields\Field>
+     */
+    public function fields(Request $request): array
+    {
+        return [
+            ID::make(__('Client ID'), 'id')->sortable(),
+            Text::make('Name', 'name')->sortable(),
+            Boolean::make('Active', 'revoked')->trueValue(0)->falseValue(1),
+            Text::make('Redirect URL(s)', 'redirect')->onlyOnDetail(),
+            Boolean::make('Public (PKCE-Enabled Client)', function (): bool {
+                return null !== $this->secret;
+            }),
+            Boolean::make('Password Client', 'password_client'),
```

Comment:
Is this referring to the password grant type?

Reply:
Yes

Reply:
I would rather pretend that doesn't exist. We could support it in theory but i have specifically put a lot of effort into not handling users' GT passwords.

Reply:
I added it for the sake of matching the model, but I'm pretty sure it wouldn't work anyways because the authentication end of it isn't implemented.

Reply:
Right, my point is to remove something that doesn't work from the UI to simplify things.